esmerald/responses/base.py

```diff
@@ -23,10 +25,10 @@
     Response as LilyaResponse,  # noqa
     StreamingResponse as StreamingResponse,  # noqa
 )
-from orjson import OPT_OMIT_MICROSECONDS, OPT_SERIALIZE_NUMPY, dumps
+from orjson import OPT_OMIT_MICROSECONDS, OPT_SERIALIZE_NUMPY, dumps, loads
 from typing_extensions import Annotated, Doc
 
-from esmerald.encoders import Encoder, json_encoder
+from esmerald.encoders import ENCODER_TYPES_CTX, Encoder, json_encoder
```

Comment:
Once LILYA_ENCODER_TYPES is rolledback, this should be changed.

Literally, why change to json_encoder when the base already was updated to use the json_encode? Rollback all of this.